lib/ioki/model/operator/payment_method.rb

```diff
@@ -5,10 +5,9 @@ module Model
     module Operator
       class PaymentMethod < Base
         attribute :type, on: :read, type: :string
-        attribute :id, on: :read, type: :string
+        attribute :id, on: [:read, :create], type: :string, omit_if_blank_on: :create
```

Comment:
That's kinda odd. The `id` should not be accessible via `create` because it is set by triebwerk. Why was it configured like that in the first place 🤔

Reply:
That makes sense. I'll remove that `:create` entirely.

Reply:
thanks 🙏

Reply:
So I think I know what's going on here: When the `PaymentMethod` is nested within another object, it already has an `:id`, so when that other object is created, the `:id` should be there. When a `PaymentMethod` is created though, it doesn't have an `:id` so we should omit it. Therefor, all these `attribute :id, on: [:read, :create], type: :string, omit_if_blank_on: :create` do make sense...

Reply:
Right, for example in `/api/passenger/rides/{id}/booking`. Thanks for catching it! We would probably have complained as soon as our tests began to fail :slightly_smiling_face:

Reply:
@phylor I added it back in and fixed a failing test. Could you please take a look now?

Reply:
Although, I guess it doesn't really make sense for the operator api 🤷

Reply:
So it's a copy and paste result? Should we remove it then?

Reply:
> So it's a copy and paste result?
Most probably. 
> Should we remove it then?
Not sure 😄

Reply:
I guess it's fine to leave it for now 👍 Thanks @eikes

Reply:
@eikes Our tests look fine! :+1: